connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -284,8 +285,33 @@ export async function syncOneFile(
             },
             `Error while converting docx document to text`
           );
-          // we don't know what to do with PDF files that fails to be converted to text.
-          // So we log the error and skip the file.
+          return false;
+        }
+      }
+    } else if (
+      file.mimeType ===
+      "application/vnd.openxmlformats-officedocument.presentationml.presentation"
+    ) {
+      if (res.data instanceof ArrayBuffer) {
+        try {
+          const converted = await PPTX2Text(Buffer.from(res.data));
+
+          documentContent = {
+            prefix: null,
```

Comment:
can we get the title here or does it get added later on?

Reply:
For PDF and Gdrive we don't get the title here, because it's added as a tag. Should we change that?

Reply:
My main point is whether the title ends-up repeated in the Content as we chunk

Reply:
Looking at the code

Reply:
It's ok we do call `renderDocumentTitleAndContent`